src/lib/ai.ts

```diff
@@ -11,11 +11,10 @@ import { makeAction, SendAction } from './bosun/action'
 import { sleep } from 'openai/core'
 
 export namespace AIEngine {
-    export type BasicModel =
-        | 'o3-mini-2025-01-31'
-        | 'o1-preview-2024-09-12'
-        | 'gpt-4o-2024-11-20'
-        | 'o1-2024-12-17'
+    /**
+     * Represents a basic model name for LLMs.
+     */
+    export type BasicModel = string
```

Comment:
Хочется сохранить автокомплит
```suggestion
    export type BasicModel = 
        | 'o3-mini-2025-01-31'
        | 'o1-preview-2024-09-12'
        | 'gpt-4o-2024-11-20'
        | 'o1-2024-12-17'
        | (string & {}) // prevents compiler from simplifying the type to just `string`
```